registrations/serializers.py

```diff
@@ -434,6 +434,25 @@ class SignUpGroupCreateSerializer(
     reservation_code = serializers.CharField(write_only=True)
     extra_info = serializers.CharField(required=False, allow_blank=True)
 
+    def validate(self, data):
+        validated_data = super().validate(data)
+
+        errors = {}
+
+        signups_data = validated_data.get("signups")
+        for signup in signups_data:
```

Comment:
If you want to save some precious lines you could;

```
if not any(signup.get("responsible_for_group") for signup in signups_data):
    errors["signups"] = _(
                "A group must have at least one participant who is responsible for the group"
            )
````

Reply:
Line saver tip applied